affiliatewp-order-details-for-affiliates.php

```diff
@@ -26,61 +26,85 @@
  * @version 1.1.5
  */
 
-// Exit if accessed directly
-if ( ! defined( 'ABSPATH' ) ) exit;
-
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+/**
+ * Main AffiliateWP_Order_Details_For_Affiliates class.
+ *
+ * @since 1.0
+ */
 final class AffiliateWP_Order_Details_For_Affiliates {
 
 	/** Singleton *************************************************************/
 
 	/**
-	 * @var AffiliateWP_Order_Details_For_Affiliates The one true AffiliateWP_Order_Details_For_Affiliates
+	 * Class instance.
+	 *
+	 * @var AffiliateWP_Order_Details_For_Affiliates
 	 * @since 1.0
 	 */
 	private static $instance;
 
-	public static  $plugin_dir;
-	public static  $plugin_url;
+	/**
+	 * Plugin directory.
+	 *
+	 * @var string
+	 * @since 1.0
+	 */
+	public static $plugin_dir;
+
+	/**
+	 * Plugin url.
+	 *
+	 * @var string
+	 * @since 1.0
+	 */
+	public static $plugin_url;
+
+	/**
+	 * Plugin version.
+	 *
+	 * @var string
+	 * @since 1.0
+	 */
 	private static $version;
 
 	/**
 	 * Order Details instance.
 	 *
-	 * @access public
-	 * @var    \AffiliateWP_Order_Details_For_Affiliates_Order_Details
+	 * @var \AffiliateWP_Order_Details_For_Affiliates_Order_Details
 	 */
 	public $order_details;
 
 	/**
 	 * Emails instance.
 	 *
-	 * @access public
-	 * @var    \AffiliateWP_Order_Details_For_Affiliates_Emails
+	 * @var \AffiliateWP_Order_Details_For_Affiliates_Emails
 	 */
 	public $emails;
 
 	/**
 	 * Shortcodes instance.
 	 *
-	 * @access public
-	 * @var    \AffiliateWP_Order_Details_For_Affiliates_Shortcodes
+	 * @var \AffiliateWP_Order_Details_For_Affiliates_Shortcodes
 	 */
 	public $shortcodes;
 
 	/**
-	 * Main AffiliateWP_Order_Details_For_Affiliates Instance
+	 * Main AffiliateWP_Order_Details_For_Affiliates Instance.
 	 *
 	 * Insures that only one instance of AffiliateWP_Order_Details_For_Affiliates exists in memory at any one
 	 * time. Also prevents needing to define globals all over the place.
 	 *
 	 * @since 1.0
-	 * @static
-	 * @staticvar array $instance
-	 * @return The one true AffiliateWP_Order_Details_For_Affiliates
+	 *
+	 * @return The one true AffiliateWP_Order_Details_For_Affiliates.
```

Comment:
```suggestion
	 * @return AffiliateWP_Order_Details_For_Affiliates The one true AffiliateWP_Order_Details_For_Affiliates.
```
Add missing return type.